contributing.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+Great that you're willing to contribute to Pylake!
+
+This document is intended to provide you with some instructions on how to contribute to Pylake.
+
```

Comment:
Good idea. I've split it up into two sections. One for issues, one for PRs.

I didn't add an explicit section on improvements to the docs in the former section, because I think those can likely directly go to a PR.